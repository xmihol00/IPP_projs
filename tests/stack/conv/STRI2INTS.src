.IPPcode21

DEFVAR GF@a
DEFVAR GF@b

PUSHS string@ABC
PUSHS int@1
STRI2INTS
POPS GF@a
WRITE GF@a
WRITE string@\010

MOVE GF@a string@0123456789
PUSHS GF@a
PUSHS int@3
STRI2INTS
POPS GF@a
WRITE GF@a
WRITE string@\010

MOVE GF@b string@0123456789
PUSHS GF@b
PUSHS int@9
STRI2INTS
POPS GF@b
WRITE GF@b
WRITE string@\010

SUB GF@a GF@b GF@a
WRITE GF@a
WRITE string@\010
