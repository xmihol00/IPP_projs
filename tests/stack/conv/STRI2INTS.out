66
51
57
6
